src/components/date-picker/date-picker.tsx

```diff
@@ -85,6 +85,12 @@ export class DatePicker {
     @Prop({ reflect: true })
     public label: string;
 
+    /**
+     * The placeholder text shown inside the input field, when the field is focused and empty
```

Comment:
Not sure about this solution here for limel-chip-set

https://github.com/Lundalogik/lime-elements/blob/0611bf4a80c989795f3c4620c2a04eabafe7e1d1/src/components/chip-set/chip-set.tsx#L397

I think we should have called `searchLabel`, `placeholder` instead. Should we deprecate it and do so? Or fall it back to placeholder?

Reply:
> normally a placeholder would always be visible as long as the field is empty,

True that! _However_, in material design, that's not the case. Labels are visible and inside the filed, when the field is empty. Therefore, even if you specify a placeholder and **no label,** the placeholder won't be shown, unless you click on the input field.

So they are working exactly the same.

Reply:
>  even if you specify a placeholder and no label, the placeholder won't be shown, unless you click on the input field.

💩 I just realized that due to this, I can't use `placeholder` as shown in my prototype here: https://github.com/Lundalogik/crm-feature/issues/2916

So to show "calculated date and time", I have to use `label` instead of `placeholder` when the filed is empty and not focused. And when it's focused, use the placeholder, and switch the label to something else :/  

Blä…

This is a good feature to have nevertheless

Reply:
Then I guess it would make sense to deprecate `searchLabel` and add `placeholder` 👍

Reply:
OK. Let's do it in another PR